tox.ini

```diff
@@ -0,0 +1,29 @@
+# For more information about tox, see https://tox.readthedocs.io/en/latest/
+[tox]
+envlist = py{37,38,39}-{linux,macos,windows}
+
+[gh-actions]
+python =
+    3.7: py37
+    3.8: py38
+    3.9: py39
+
+[gh-actions:env]
+PLATFORM =
+    ubuntu-latest: linux
+    macos-latest: macos
+    windows-latest: windows
+
+[testenv]
+platform =
+    macos: darwin
+    linux: linux
+    windows: win32
```

Comment:
Why test on Windows 32bit?

Reply:
good question! will change to `windows` -- this will prob be true for the other plugins too.

Reply:
done!